docs/source/api/python/techniques/ISISPowder-Polaris-v1.rst

```diff
@@ -263,25 +263,27 @@ Example Input:
   # Notice the filename always ends in .yaml
   polaris_example = Polaris(calibration_mapping_file=r"C:\path\to\file\calibration_mapping.yaml", ...)
 
-.. _chopper_on_polaris_isis-powder-diffraction-ref:
+.. _mode_polaris_isis-powder-diffraction-ref:
 
-chopper_on
+mode
 ^^^^^^^^^^
-The chopper state to use in the 
+The current chopper mode to use in the 
 :ref:`create_vanadium_polaris_isis-powder-diffraction-ref`
 and :ref:`focus_polaris_isis-powder-diffraction-ref` method.
 This determines which vanadium and empty run numbers
 to use whilst processing.
 
 Accepted values are: **True** or **False**
 
```

Comment:
Shouldn't accepted values now be `PDF` or `Rietveld` now that you have changed it?